packages/evm/contracts/test/arbitration/Arbitrable.sol

```diff
@@ -22,8 +22,15 @@ contract ArbitrableMock is IArbitrable {
         if (_finished) arbitrator.closeEvidencePeriod(_disputeId);
     }
 
+    function submitEvidenceFor(uint256 _disputeId, address _submitter, bytes calldata _evidence, bool _finished) external {
+        arbitrator.submitEvidence(_disputeId, _submitter, _evidence);
+        if (_finished) arbitrator.closeEvidencePeriod(_disputeId);
+    }
+
     function rule(uint256 _disputeId) external {
         (, uint256 ruling) = arbitrator.rule(_disputeId);
-        emit Ruled(arbitrator, _disputeId, ruling);
+        if (ruling > 0) {
```

Comment:
Can you get 0 back? I always thought it was 2, 3, or 4

Reply:
Not really. Addressed in 3dc65487976aaacb8864552bab8211e7cccffcc7